using_openshift/builds.adoc

```diff
@@ -37,3 +37,115 @@ Build can be manually canceled using following command:
 ----
 osc cancel-build <buildID>
 ----
+
+== Build triggers
+When defining a BuildConfig, you can define triggers to control circumstances under which a Build should be run for the BuildConfig.  There are 2 main types of triggers available:
+
+* Webhook
+* Image change
+
+=== Webhook triggers
+Webhook triggers allow you to trigger a new build by sending a request to the OpenShift API endpoint.  You can define two types of webhook triggers:  
+
+* https://developer.github.com/webhooks/[Github Webhooks]
+* Generic Webhooks
+
+==== Github webhooks
+Github webhooks can handle the call made by Github when a repository is updated.  When defining the trigger you can specify a `secret` which you will supply to Github when configuring the webhook.  This will ensure that only you and your repository can trigger the build.  The trigger definition json within the BuildConfig looks like this:
```

Comment:
I think the generic hook wording is probably ok as is.  you do have to provide the secret when making the request (in the url).  close enough imho.